src/Sylius/Bundle/CoreBundle/Twig/ProductVariantsMapExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Twig;
+
+use Sylius\Bundle\CoreBundle\Templating\Helper\ProductVariantsMapHelper;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+final class ProductVariantsMapExtension extends AbstractExtension
+{
+    public function __construct(private ProductVariantsMapHelper $productVariantsMapHelper)
```

Comment:
Do we need this helper here? We can't use the provider directly? It looks to me like an unnecessary additional service, especially as there is no logic there other than calling the provider 🤔